params/config.libevm.go

```diff
@@ -231,3 +205,48 @@ func (r *Rules) extraPayload() *pseudo.Type {
 	}
 	return r.extra
 }
+
+func (c *ChainConfig) extraDescription() string {
+	if registeredExtras == nil || c.extra == nil {
+		return ""
+	}
+
+	type Descriptioner interface {
+		Description() string
+	}
+	d, ok := c.extra.Interface().(Descriptioner)
```

Comment:
(no action required) I'm going to remove `pseudo.Type.Interface()` because it's only used in tests and should be replaced with `pseudo.Type.Equals()`.

Reply:
already gone.